<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1081</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1051</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>华文隶书</family>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>数字图像处理器</string>
        </property>
        <property name="textFormat">
         <enum>Qt::MarkdownText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="PushButton" name="pushButton_10">
             <property name="text">
              <string>图像选择</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>图像二值化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>图像旋转</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>图像锐化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>图像加噪</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>图像滤波</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>边缘检测</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>亮度调节</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>素描效果</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="text">
              <string>马赛克</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="9,1">
              <item>
               <widget class="QWidget" name="widget_7" native="true"/>
              </item>
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <widget class="QSlider" name="horizontalSlider">
                 <property name="geometry">
                  <rect>
                   <x>120</x>
                   <y>20</y>
                   <width>160</width>
                   <height>22</height>
                  </rect>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
                <widget class="QLabel" name="label_2">
                 <property name="geometry">
                  <rect>
                   <x>50</x>
                   <y>20</y>
                   <width>53</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>系数</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="9,1">
              <item>
               <widget class="QWidget" name="widget_9" native="true">
                <widget class="QLabel" name="label_5">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>10</y>
                   <width>441</width>
                   <height>491</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_10" native="true">
                <widget class="QLabel" name="label_3">
                 <property name="geometry">
                  <rect>
                   <x>9</x>
                   <y>9</y>
                   <width>61</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>方法简介</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_4">
                 <property name="geometry">
                  <rect>
                   <x>74</x>
                   <y>9</y>
                   <width>381</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>该方法通过设置阈值，设置为0或1</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header location="global">qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
